src/main/resources/view/DarkTheme.css

```diff
@@ -196,6 +196,18 @@
     -fx-text-fill: #010504;
 }
 
+.info_big_label {
+    -fx-font-family: "SF Mono Semibold";
```

Comment:
Is it possible to use an alternative font as a backup in case the users' machine are unable to render the font style?

Reply:
I don't think this is necessary as the font comes along with the `.jar` file which means the application is always able to load the font.